README.md

```diff
@@ -44,6 +44,42 @@ is installed:
 - Select "Debug" or "Release" at the top.
 - Hit F5 to build. The output will be in the "bin/Release" or "bin/Debug" directory.
 
+# Building (macOS)
+All the commands should be executed in terminal. All global package installs should be done only if not yet installed.
+
+1. Install Homebrew, a package manager for macOS (if not yet installed): Follow instructions on https://brew.sh/
+
+2. Install Mono (https://www.mono-project.com/):
+`brew install mono`
+
+3. Install .NET Core SDK (https://dotnet.microsoft.com/download):
+`brew install dotnet-sdk`
```

Comment:
I dont see any dotnet core command.
Why do you need netcore if you use mono?

Reply:
Yeah, you absolutely right. I installed it at the testing stage and it was in my notes since then.

I updated the README file and removed that unnecessary install of .NET Core SDK.